_episodes/02-CMSDataAnalysisSchoolPreExerciseSecondSet.md

```diff
@@ -1,19 +1,363 @@
-- ""
+- "How to slim a MiniAOD file?"
+- "How to know the size of a MiniAOD file?"
+- "How to use FWLite to analyze data and MC?"
 objectives:
-- ""
+- "Learn how to reduce the size of a MiniAOD by only keeping physics objects of interest."
+- "Learn how to determine the size of a MiniAOD file using EDM standalone utilities"
+- "Learn to use FWLite to perform simple analysis."
 keypoints:
-- ""
+- "A MiniAOD file can be `slimmed` by just retaining physics objects of interest."
+- "EDM standalone utilities can be used to determine the size of MiniAOD files."
+- "FWLite is a useful tool to perform simple analysis on a MiniAOD file."
 ---
 
 # Introduction
+Welcome to the second set of CMSDAS pre-exercises. As you know by now, the purpose of the pre-workshop exercises is for prospective workshop attendees to become familiar with the basic software tools required to perform physics analysis at CMS before the workshop begins. Post the answers in the online response form available from the **course web area:**
 
-# Exercise 1 - <FIXME>
+> ## Indico page
+>[CMSDAS pre-exercises indico page](https:/indico.cern.ch/event/966368/)
+>
+{: .callout} 
 
+The Second Set of exercises begins with `Exercise 7` . We will use Collision data events and simulated events (Monte Carlo (MC)). To comfortably work with these files, we will first make them smaller by selecting only the objects that we are interested in (electrons and muons in our case)
 
+The collision data events are stored in `DoubleMuon.root`. **DoubleMuon** refers here to the fact, that when recording these events, we believed that there are two muons in the event. This is true most of the time, but other objects can fake muons, hence at closer inspection we might find events that actually don't have two muons.
+
+The MC file is called **DYJetsToLL**. You will need to get used to cryptic names like this if you want to survive in the high energy physics environment! The MC file contains Drell Yan events, that decay to two leptons and that might be accompanied by one or several jets. 
+
+`Exercises 8` and `Exercise 9` are using FWLite (**Frame Work Lite**). This is an interactive analysis tool integrated with the CMSSW EDM (Event Data Model) Framework. It allows you to automatically load the shared libraries defining CMSSW data formats and the tools provided, to easily access parts of the event in the EDM format within ROOT interactive sessions. It reads produced ROOT files, has full access to the class methods and there is no need to write full-blown framework modules. Thus having FWLite distribution locally on the desktop one can do CMS analysis outside the full CMSSW framework. In these two exercises, we will analyze the data stored in a MiniAOD sample using FWLite. We will loop over muons and make a Z mass peak.
+
+**We assume that having done the first set of pre-exercises by now, one is comfortable with logging onto** `cmslpc-sl7.fnal.gov` **and setting up the cms environment.**
+
+# Support
```

Comment:
As you can see from the other exercises, we didn't include the Support, Support email, mattermost, or Obtain a ... sections. All of this is included in the setup directions. It's better to have one authoritative spot for all of those directions. Make sure what is in the setup.md document is good enough for you and then remove these sections from this document.